src/settings/command_line.rs

```diff
@@ -11,6 +11,9 @@ use crate::error::{usage_error, ShellError};
 #[derive(Debug, StructOpt, Default)]
 #[structopt(name = "qldb", about = "A shell for interacting with Amazon QLDB.")]
 pub struct Opt {
+    /// The AWS Region code of the QLDB ledger to connect to. For example: `us-east-1`.
+    /// By default, the shell will pick a default region as described in the 
+    /// standard AWS toolchain [documentation](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-configure.html).
```

Comment:
I don't think markdown links are what you want here. This text is going to be rendered verbatim when the user types `--help`.

I think we should just put the links in the shell guide, and link the shell guide in the help (line 12).

Reply:
Thanks, removed the markdown links and added the shell guide in the help